css/style.css

```diff
@@ -1,3 +1,69 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
 body {
-  background: wheat;
+  background-color: rgb(237, 242, 248);
+}
+h1,
+h2 {
+  margin: 0;
+}
+.card {
+  padding: 1.5em 2em 2.5em;
+  background-color: rgb(115, 63, 199);
+  border-radius: 0.5em;
+  margin: 20px;
+}
+img {
+  border-radius: 50%;
+  border: 3px solid rgb(167, 117, 242);
+}
```

Comment:
never add border radius to the `img tag`, also add max-width to 100% and make it display block